src/app/features/users-page/users-page.component.html

```diff
@@ -1,13 +1,13 @@
 <a routerLink="/" class="back-link">Back To Home</a>
 <h1>List of all users:</h1>
-<table style="width: 1000px; border: 1px solid black">
+<table class="table-wrapper">
     <tr>
         <th>NAME</th>
     </tr>
     <tbody *ngIf="users$ | async as users">
         <tr *ngFor="let user of users">
             <th>
-                <a [routerLink]="['/profile/' + user.firstName]">{{ user.firstName }}</a>
+                <a [routerLink]="['/profile/' + user.id]">{{ user.firstName }}</a>
```

Comment:
`UsersPageComponent` has default change detection strategy it means this expression `'/profile/' + user.id` will be recalculated on each app tick.
This is not good.
For components with default CD strategy you need to prevent using expressions inside templates.
You have multiple options to workaround it
Here you have a list of items so the best way is to add `trackBy` function to not rerender each users template

Reply:
@maaxnikitin94 fix it please